### DISCUSSION TITLE
Multi-account support

### DISCUSSION THREAD

**Discussion by mskelton** [zoidsh/meru#243] [Feature Requests] [answered]
Currently, multi-account support is limited to signing into another Google account using the account selector in the top right, but we should explore better multi-account support.

**Comment by bsramin** [comment]
use this? https://github.com/brrd/electron-tabs

**Comment by timche** [comment]
> use this? https://github.com/brrd/electron-tabs

Looks good 👍

**Comment by gurumark** [comment]
Could you provide clear instructions on how to use electron-tabs?
Thanks.

**Comment by mskelton** [comment]
> Could you provide clear instructions on how to use electron-tabs?

@gurumark I think that is kind of up to whoever ends up doing the PR for this.

**Comment by mskelton** [comment]
Not sure how electron tabs will work since we have our own custom header bar.  We may need to consider alternate methods of handling multiple accounts.

**Comment by floriangbh** [comment]
I'm also waiting for multi-account support (but not skilled in TS/JS hehe) !

Thanks for your work !

**Comment by timche** [comment]
I'm planning to work on this using React and https://electronjs.org/docs/api/browser-view.

**Comment by brahimmouhamou** [comment]
What is the status of this feature? 
I'm also willing to test (macOS Catalina 10.15.6)

**Comment by duongleh** [comment]
This is the only missing feature that prevents me from using this app as a daily mail client. Can't wait for it.

**Comment by timche** [comment]
There is something in the works. Stay tuned!

**Comment by gurumark** [comment]
What is the max number of accounts one can sign in at the same time?

**Reply by timche** [reply]
There is currently no limit, but too many accounts will definitely use more hardware resources and can potentially become unresponsive.

**Comment by timche** [comment, marked as answer]
Multi-accounts is coming with v3 (soon). If you have any questions, please ask them in #242.
